tools/ci-build/runtime-versioner/src/command/audit.rs

```diff
@@ -21,6 +21,9 @@ use std::{
     process::Command,
 };
 
+const SMITHY_RS_RUNTIME_VERSIONER_AUDIT_PREVIOUS_RELEASE_TAG: &str =
```

Comment:
**question**: Why do we need an env variable when the arguments include [previous_release_tag](https://github.com/smithy-lang/smithy-rs/blob/69efe3a72867b0f269ee840f8042f5f477d7476a/tools/ci-build/runtime-versioner/src/main.rs#L34). 

Also can we not make use of the [env](https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes) attribute so that `clap` sources this for us into the `Audit` command line arguments?

Reply:
During development, we usually run `runtime-versioner` as [part of pre-commit hooks](https://github.com/smithy-lang/smithy-rs/blob/main/.pre-commit-hooks/runtime-versioner.sh#L8). Do you know if there is a way to specify `--previous-release-tag <some release tag>` to the above invocation done by pre-commit hooks?

> Also can we not make use of the [env](https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes) attribute

Haven't used this feature before. Let me try that.